app/bundles/EmailBundle/Controller/Api/EmailApiController.php

```diff
@@ -214,4 +215,13 @@ public function replyAction(Reply $replyService, RandomHelperInterface $randomHe
             $this->view(['success' => true], Response::HTTP_CREATED)
         );
     }
+
+    protected function prepareParametersFromRequest(FormInterface $form, array &$params, object $entity = null, array $masks = [], array $fields = []): void
+    {
+        if (isset($params['publicPreview']) && $entity instanceof Email) {
+            $entity->setPublicPreview(InputHelper::boolean($params['publicPreview']) ?? false);
```

Comment:
See https://github.com/mautic/mautic/pull/13573#pullrequestreview-1983174909

Reply:
I've answer there. I copy it again. Code seems correct to me:

> Based on code, isset($params['publicPreview']) is in parent condition and this InputHelper::booleen in some cases can return null. Then it's correct, always set bool in setPublicPreview method.

Reply:
I missed that. Makes sense.